src/content/docs/ai-monitoring/view-ai-data.mdx

```diff
@@ -0,0 +1,187 @@
+* **AI entities**: View a table summary of all entities reporting AI data. See entities with standard APM data like error rate, throughput, and app response time. When you select an entity, you can start exploring the APM **AI responses** page. 
+* **Compare models**: Compare token usage, response time, and error rate between different models. If you're conducting A/B tests, get all the information you need to make decisions about your AI-powered app.
+
+## AI responses page [#ai-responses]
+
+The top-level **AI responses** page shows your AI data in aggregate. Aggregated data takes the average total responses, response times, and token usage per response across all entities reporting AI data. On this page, response refers to an output from your AI-powered app when given a prompt. 
+
+If you own several apps with various implementations of different AI frameworks, you can get a general sense for how your AI models perform.  
+
+### Track total responses, average response time, and token usage  
+
+<img
+    title="AI responses response billboard and timeseries graphs"
+    alt="A cropped screenshot displaying the timeseries graphs and billboard info about AI data"
+    src={aiTimeseriesBillboard}
+/>
+
+The three tiles show general performance metrics about your AI's responses. These tiles may not tell you the exact cause behind a problem, but they're useful for identifying anomalous fluctuations in your app's performance. 
+
+<img
+    title="Three tiles to show spikes and drops in response usage"
+    alt="A cropped screenshot displaying billboard info about AI data"
+    src={aiCroppedImageofAIBillboards}
+/>
+
+* If you notice a drop in total responses or increase in average response time, it can indicate that some technology in your AI toolchain has malfunctioned and prevented your AI-powered app from posting a response.  
+* A drop or increase in average token usage per response can give you insight into how your model creates a response. Maybe it's pulling too much context, thus driving up token cost while generating its response. Maybe its responses are too spare, leading to lower token costs and unhelpful responses.
+
+### Adjust the timeseries graphs 
+
+<img
+    title="AI timeseries graphs"
+    alt="A cropped screenshot displaying timeseries info about AI data"
+    src={aiCroppedImageofAItimeseries}
+/>
+
+You can refer to the time series graphs to better visualize when an anamolous behavior first appears. 
+
+* Adjust the timeseries graph by dragging over a spike or drop. This scopes the timeseries to a certain time window.
+* Select the drop down to run comparative analysis for different performance parameters. You can choose between total responses, average response time, or average tokens per response. 
+* If you've enabled the feedback feature, you can scope the graphs to analyze responses by positive and negative feedback. 
```

Comment:
Possible to link to wherever the feedback features is documented, or a link to it in the product or something like that?